pkg/loadbalancer/experimental/bpf_reconciler.go

```diff
@@ -601,6 +610,31 @@ func (ops *BPFOps) updateFrontend(fe *Frontend) error {
 		}
 	}
 
+	// Update Maglev
+	if ops.cfg.NodePortAlg == option.NodePortAlgMaglev {
```

Comment:
Perhaps move this into a helper as it's fair bit of lines and this method is getting big?

Reply:
Done.